dom.bs

```diff
@@ -3040,6 +3044,44 @@ standards that want to define APIs shared between <a for=/>documents</a> and
  <a for=CharacterData>data</a> is the string and <a for=Node>node document</a> is
  <var>document</var>.
 
+ <li><p>Let <var>isScriptElement</var> be true if <var>parent</var> is an {{HTMLScriptElement}}
+ <a for=/>node</a>; otherwise false.
+
+ <li>
+  <p><a for=set>For each</a> <var>value</var> of <var>nodes</var>:
+
+  <ol>
+   <li>
+    <p>If <var>value</var> is a <a for=/>node</a>, then:
+
+    <ol>
+     <li><p>If <var>isScriptElement</var> is false, then <a for=iteration>continue</a>.
+
+     <li><p>If <var>value</var> is not a {{Text}} <a for=/>node</a>, then <a for=iteration>continue</a>.
+
+     <li><p>Let <var>text</var> be the result of calling <a abstract-op>Get Trusted Type compliant string</a>,
+     with {{TrustedScript}}, <var>document</var>'s <a>relevant global object</a>,
+     <var>value</var>'s <a for=CharacterData>data</a>, "<code>HTMLScriptElement text</code>",
+     "<code>script</code>".
+
+     <li><p>Set <var>value</var>'s <a for=CharacterData>data</a> to <var>text</var>.
```

Comment:
Why define _text_ if we use it once? I would inline this step into the one above.

Reply:
In fact, would this work:

> If _value_ is a Text node and isScriptElement is true, then set _value_'s data to ...

?